webapp-django/crashstats/home/jinja2/home/home.html

```diff
@@ -86,10 +46,30 @@ <h4>{{product}} {{version}}</h4>
                                 version=version,
                                 process_type='plugin',
                                 days=days,
```

Comment:
This shouldn't be passing "days" as an argument. Ditto for the other links. I'll fix that.